tests/system/apps/__init__.py

```diff
@@ -99,3 +99,7 @@ def ipv6_healthcheck(app_id=None, parent_group="/"):
         For more information about the nc options just run `docker run alpine nc --help`
     """
     return load_app('ipv6-healthcheck', app_id, parent_group)
+
+
+def readiness_ssl(app_id=None, parent_group="/"):
```

Comment:
nit: I'm not quite happy with the name. How about `app-with-https-readiness-checks`?

Reply:
done